pkgs/applications/version-management/git-and-tools/subgit/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation {
   '';
 
   src = fetchurl {
-    url = http://old.subgit.com/download/subgit-3.1.0.zip;
-    sha256 = "08qhpg6y2ziwplm0z1ghh1wfp607sw4hyb53a7qzfn759j5kcdrg";
+    url = "http://subgit.com/download/${name}.zip";
+    sha256 = "13r6hi2zk46bs3j17anfc85kszlwliv2yc16qx834b3v4w68hajw";
   };
-}
```

Comment:
There is a no newline at end of file error here.

Can you change buildInputs to nativeBuildInputs as the pkgs used is not needed for runtime